src/blueprints/Do.Blueprints.Service.Application/CodingStyle/UseNullableTypes/NonOptionalNotNullParametersAreRequiredConvention.cs

```diff
@@ -15,4 +15,13 @@ public void Apply(ParameterModelContext context)
 
         context.Parameter.AddRequiredAttributes();
     }
+
+    public void Apply(ApiModelContext context)
+    {
+        context.Api.Usings.AddRange([
+            "Microsoft.AspNetCore.Mvc.ModelBinding",
+            "Newtonsoft.Json",
+            "System.ComponentModel.DataAnnotations"
+        ]);
```

Comment:
move all usings to their feature under `ConfigureApiModel` instead of `ConfigureApiModelConventions`